salt/modules/artifactory.py

```diff
@@ -465,7 +468,7 @@ def __save_artifact(artifact_url, target_file, headers):
     try:
         request = urllib.request.Request(artifact_url, None, headers)
         f = urllib.request.urlopen(request)
-        with salt.utils.files.fopen(target_file, "wb") as local_file:
+        with salt.utils.fopen(target_file, "wb") as local_file:
```

Comment:
Why change this back? `salt.utils.fopen` is the preference here and `salt.utils.fopen` is being deprecated.